packages/xo-web/src/common/intl/messages.js

```diff
@@ -1023,6 +1023,8 @@ const messages = {
   // ----- VM advanced tab -----
   vmRemoveButton: 'Remove',
   vmConvertToTemplateButton: 'Convert to template',
+  vmVirtualizationPVToHVM: 'Switch from PV to HVM',
```

Comment:
I think it's unnecessary to display from which mode we want to convert, simply *Convert to HVM*.